docker/sorry-cypress/SETUP.md

```diff
@@ -0,0 +1,24 @@
+# Setup Sorry Cypress
+
+1. Use the docker compose file (called `sorry-cypress.yml`) in this folder as base.
+2. Replace the `<insert-minio-access-key>` and `<insert-minio-secret>` with applicable values (e.g random). These values are needed to manage the minio instance. 
+3. Set a random key for the `ALLOWED_KEYS` (by replacing `<insert-cypress-key>`). This key is needed to authorize bamboo against the sorry-cypress dashboard. 
+4. Place all the necessary NGINX config files in their locations
+    1. Place the `nginx.conf` file from this folder into this path `files/nginx/nginx.conf`
+    2. Place a generated `.htpasswd` file into this path `files/nginx/.htpasswd`
+    3. Place/link a public SSL certificate file in this path `files/nginx/fullchain.pem`
+    4. Place/link a private SSL certificate file in this path `files/nginx/privkey.pem`
+5. Ensure that all the URLs within the `nginx.conf` and the `sorry-cypress.yml` file are still valid
+6. Start the containers by using `docker compose up -f sorry-cypress.yml`
+7. Login into the minio dashboard (https://minio.sorry-cypress.ase.cit.tum.de)
```

Comment:
Does Minio have a REST API to configure all of this? If yes, we could create a script to do this automatically.
If this is possible lets discuss this via Zoom.

Reply:
There is a cli for it, but not all things can be done via the cli. And when you have to do some things in the UI, I think it is feasible to also do the other simple steps in the UI. It takes like 1 minute to do that and I think that we will not setup a sorry cypress instance that often anyway